src/fieldAnalyser/typeUtils.ts

```diff
@@ -0,0 +1,39 @@
+import {FieldTypes} from '@coveord/platform-client';
+
+const acceptedTransitions: Partial<Record<FieldTypes, FieldTypes[]>> = {
+  [FieldTypes.LONG]: [FieldTypes.LONG_64, FieldTypes.DOUBLE],
+  [FieldTypes.LONG_64]: [FieldTypes.DOUBLE],
+};
```

Comment:
Mh, I would do this as an `Array<{from:FieldTypes, to:Array<FieldTypes>}>`. Verbose, but more readable I reckon.